.github/workflows/check.yml

```
@@ -13,14 +13,25 @@ permissions:
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install pyright
+        run: npm install -g pyright
+      - name: Run pyright on server
+      - uses: jakebailey/pyright-action@v2
   build-android-app:
     runs-on: ubuntu-latest
     timeout-minutes: 30
+    working-directory: client
```

actionlint (1.7.4)

unexpected key "working-directory" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)